renovate.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "description": "Renovate configuration for expirable storage repository",
+  "extends": [
+    "config:base",
+    ":dependencyDashboard",
+    ":rebaseStalePrs"
+  ],
+  "schedule": [
+    "before 3am every weekday"
+  ],
+  "enabledManagers": [
+    "github-actions",
+    "npm"
+  ],
+  "packageRules": [
```

Comment:
Based on contents of https://github.com/wayfair-incubator/expirable-storage/blob/main/package.json, I'd suggest creating additional package groups for `typescript` ([example here](https://github.com/wayfair-incubator/forker/blob/main/renovate.json#L37-L38)) and `webpack/webpack-*` ([example here](https://github.com/wayfair-incubator/auxeng-docs/blob/main/renovate.json#L38-L40)) since those are both highly likely to introduce breaking changes to JS/TS projects. We can then enforce `allowedVersions` for each if needed later.

Reply:
Thanks for reviewing, requested changes are added to this PR!